GenderPayGap.WebUI/Views/AdminOrganisationStatus/ChangeStatus.cshtml

```diff
@@ -0,0 +1,149 @@
+
+            @Html.GovUkErrorSummary(new[]
+            {
+                Model.CurrentStatus == OrganisationStatuses.Active ? nameof(AdminChangeStatusViewModel.NewStatusFromActive) : Model.CurrentStatus == OrganisationStatuses.Retired ? nameof(AdminChangeStatusViewModel.NewStatusFromRetired) : nameof(AdminChangeStatusViewModel.NewStatusFromDeleted),
+                nameof(AdminChangeStatusViewModel.Reason)
+            })
+
+            <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Organisation name</th>
+                        <td class="govuk-table__cell">@(Model.OrganisationName)</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Current status</th>
+                        <td class="govuk-table__cell">@(Model.CurrentStatus)</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <div class="govuk-form-group">
+                @if (Model.CurrentStatus == OrganisationStatuses.Active)
+                {
+                    @Html.GovUkRadiosFor(
+                        m => m.NewStatusFromActive,
+                        new FieldsetViewModel
+                        {
+                            Legend = new LegendViewModel
+                            {
+                                Text = "Please select a new status",
+                                Classes = "govuk-fieldset__legend--m"
+                            },
+                            Classes = "govuk-!-margin-top-6"
+                        })
+                }
+                else if (Model.CurrentStatus == OrganisationStatuses.Retired)
+                {
+                    @Html.GovUkRadiosFor(
+                        m => m.NewStatusFromRetired,
+                        new FieldsetViewModel
+                        {
+                            Legend = new LegendViewModel
+                            {
+                                Text = "Please select a new status",
+                                Classes = "govuk-fieldset__legend--m"
+                            },
+                            Classes = "govuk-!-margin-top-6"
+                        })
+                }
+                else
+                {
+                    @Html.GovUkRadiosFor(
+                        m => m.NewStatusFromDeleted,
+                        new FieldsetViewModel
+                        {
+                            Legend = new LegendViewModel
+                            {
+                                Text = "Please select a new status",
+                                Classes = "govuk-fieldset__legend--m"
+                            },
+                            Classes = "govuk-!-margin-top-6"
+                        })
+                }

```

Comment:
This might be a nice place to show the explanatory / hint text.
I'm thinking we could use this pattern:
https://design-system.service.gov.uk/components/radios/#radio-items-with-hints

`@Html.GovUkRadioButtonsFor()` doesn't support hints, but we could use the `@Html.GovUkRadios()` method and just provide the details.
This would then mean we had full control over what radios were shown, so we wouldn't need to have 3 separate Enums, which would make things clearer.

What do you think?